projects links, updated about text

jbyyny opened the pull request:
added PDF button link, cleaned up about and resume text, added links to
more projects

Commit:
projects links, updated about text

added PDF button link, cleaned up about and resume text, added links to
more projects

annawchou commented on resume.html:
```diff
@@ -25,10 +25,10 @@ <h1 class="line">Resume</h1>
                     <br/>
 
-                    <button><a href="#">Download PDF</a></button>
+                    <button><a href="images/Jen-Byyny-resume.pdf" target='blank'>Download PDF</a></button>
```
sorry, i made a mistake here before. it should be `<a href="images/Jen-Byyny-resume.pdf" target='blank'><button>Download PDF</button></a>` so that the link wraps the entire button.

Commit:
fixed button code

<a href="images/Jen-Byyny-resume.pdf" target='blank'><button>Download
PDF</button></a>
